notebooks/Manage Map Areas with Group and Index.ipynb

```diff
@@ -15,10 +15,7 @@
     ">\n",
     ">Source: More info: https://doc.arcgis.com/en/arcgis-online/manage-data/take-maps-offline.htm\n",
     "\n",
-    "This process assumes that you are the owner of web map and that you are using either Enterprise 11.4+ or ArcGIS Online. \n",
-    "\n",
-    "This script can be updated to run on older versions of Enterprise (pre 2.4 ArcGIS API for Python). You would need to update `from arcgis.map import Map` to `from arcgis.mapping import webmap` along with the where webmap is referenced in code. \n",
-    "More details on ArcGIS API for Python 2.4: https://www.esri.com/arcgis-blog/products/api-python/announcements/whats-new-in-arcgis-api-for-python-2-4-0-blog/\n",
+    "This process assumes that you are the owner of web map and that you are using either Enterprise 11.3 or earlier.\n",
```

Comment:
I think I would mention the python api version here, as that is really what is driving the compatibility